web/src/services/SpanAttribute.service.ts

```diff
@@ -37,10 +37,11 @@ const getCustomAttributeList = (attributeList: TSpanFlatAttribute[]) => {
 };
 
 const SpanAttributeService = () => ({
+  additionalAttributes: (count: number): TSpanFlatAttribute[] => [{key: 'tracetest.selected_spans.count', value: count.toString()}],
```

Comment:
Let's add the key name to the constants file that contains all of the supported pseudo attributes

Reply:
Do you mind changing the function name to be more descriptive like:

getPseudoAttributeList 

As we are calling them pseudo attributes